### PR TITLE
[docs] Review of the text part: Changing clusterDomain without downtime

### PR DESCRIPTION
**Pull request by Lada7878**
## Description

Text review of the https://github.com/deckhouse/deckhouse/pull/9739.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

```changes
section: node-manager, control-plane-manager, kube-dns
type: chore
summary: Minor documentation updates.
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Lada7878. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Lada7878. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Lada7878. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mortelumina98. I've started the workflow for label 'deploy/web/stage'...

File: modules/040-node-manager/openapi/config-values.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ properties:
     description: |
       Set to 'true' to disable machine-controller-manager in case of potentially cluster-damaging changes.
 
-      > **Warning!** Use this flag only if you know what you are doing and clearly understand the consequences.
+      > **Warning!** Use this flag only if you know why you are doing and clearly understand the consequences.
```
Comment (z9r5) [inline modules/040-node-manager/openapi/config-values.yaml:26]:
```suggestion
      > **Warning!** Use this flag only if you know why you are doing it and clearly understand the consequences.
```

File: modules/042-kube-dns/docs/FAQ.md [hunk 2]
```diff
@@ -5,63 +5,60 @@ search: DNS, domain, clusterdomain
...
+**Important!** If your Kubernetes version is 1.20 and higher, your controllers in the cluster use [advanced ServiceAccount tokens](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection) to work with API-server. Those tokens have extra fields `iss:` and `aud:` that contain `clusterDomain` (e.g. `"iss": "https://kubernetes.default.svc.cluster.local"`). After changing the clusterDomain, the API-server will start issuing tokens with the new service-account-issuer, but thanks to the configuration of additionalAPIAudiences and additionalAPIIssuers, the apiserver will continue to accept the old tokens.
```
Comment (z9r5) [inline modules/042-kube-dns/docs/FAQ.md:61]:
```suggestion
**Important!** If your Kubernetes version is 1.20 and higher, your controllers in the cluster use [advanced ServiceAccount tokens](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection) to work with API server. Those tokens have extra fields `iss:` and `aud:` that contain `clusterDomain` (e.g. `"iss": "https://kubernetes.default.svc.cluster.local"`). After changing the clusterDomain, the API server will start issuing tokens with the new service-account-issuer, but thanks to the configuration of additionalAPIAudiences and additionalAPIIssuers, the apiserver will continue to accept the old tokens.
```

File: modules/040-control-plane-manager/openapi/doc-ru-config-values.yaml [hunk 3]
```diff
@@ -9,14 +9,14 @@ properties:
         properties:
           issuer:
             description: |
-              Издатель ServiceAccount'ов. Это URL API-сервера. Значения этого поля используются как `iss` claim токена и для проверки JWT-токенов ServiceAccount.
+              Список API дополнительных издателей токенов ServiceAccount'ов, которые нужно включить при создании таких токенов. Это URL API-сервера. Значения этого поля используются как `iss` claim токена и для проверки JWT-токенов ServiceAccount.
```
Comment (z9r5) [inline modules/040-control-plane-manager/openapi/doc-ru-config-values.yaml:12]: @Lada7878 Тут ошибка? Это не дополнительные

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Lada7878. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Lada7878. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Lada7878. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Lada7878. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Lada7878. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [10465](https://github.com/deckhouse/deckhouse/pull/10465) to the branch [release-1.65](https://github.com/deckhouse/deckhouse/tree/release-1.65) successful!

**Pull request merged**